Libraries Usage/RxMoyaExample/Podfile

```diff
@@ -1,12 +1,23 @@
-platform :ios, '8.0'
+platform :ios, '9.0'
 use_frameworks!
 
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['SWIFT_VERSION'] = '3.0'
+      config.build_settings['MACOSX_DEPLOYMENT_TARGET'] = '10.10'
+    end
+  end
+end
+
 target 'RxMoyaExample' do
 
-pod 'Moya/RxSwift'
-pod 'Moya-ModelMapper/RxSwift'
-pod 'RxSwift', '~> 2.0'
-pod 'RxCocoa', '~> 2.0'
-pod 'RxOptional', '~> 2.0'
+pod 'RxSwift',    '~> 3.0.0-beta.2'
```

Comment:
Because now RxSwift is officially 3.0.0, can we try with 3.0.0 versions of RxSwift & RxCocoa? :)